f8a_worker/utils.py

```diff
@@ -690,3 +691,28 @@ def get_user_email(user_profile):
         return user_profile.get('email', default_email)
     else:
         return default_email
+
+
+def get_response(url, headers=None, sleep_time=2, retry_count=10):
+    """
+    Simple wrapper for requests which tries to get response
+    :param url: URL where to do the request
+    :param headers: additional headers for request
+    :param sleep_time: sleep time between retries
+    :param retry_count: number of retries
+    :return: content of response's json
+    """
```

Comment:
According to pep for docstring, this should be:
```python
   """Simple wrapper for requests which tries to get response.

   :param url: URL where to do the request
   :param headers: additional headers for request
   :param sleep_time: sleep time between retries
   :param retry_count: number of retries
   :return: content of response's json
   """
```

(If we want to start using proper docstrings that pass checks of tools such as pydocstyle).